src/components/heatmap/HeatmapSubscriber.js

```diff
@@ -201,12 +205,17 @@ export default function HeatmapSubscriber(props) {
         setIsRendering={setIsRendering}
         setCellHighlight={setCellHighlight}
         setGeneHighlight={setGeneHighlight}
+        setTrackHighlight={setTrackHighlight}
         setComponentHover={() => {
           setComponentHover(uuid);
         }}
         updateViewInfo={setComponentViewInfo}
         observationsTitle={observationsTitle}
         variablesTitle={variablesTitle}
+        variablesDashes={false}
+        observationsDashes={false}
+        cellColorLabels={['Cell Set']}
```

Comment:
Why not use the name of the clustering?  Name could get long though so this might be too much....at the minimum, using "Cell" hardcoded seems like we'd be asking for more work down the line to allow users to change it.

Reply:
It is a good point about not using "cell" when we have the observationLabelOverride props. I think there are two issues about using the name of the clustering - 1 that we would have to account for the string being too long and 2 that we do not have names for the "levels" of the hierarchy. For instance we would only be able to say something like "Cell Type Annotations Level 1" and "Cell Type Annotations Level 2" for the codeluppi example. I will keep it "{obs} Sets" for now but we can revisit later if we need to.